bftengine/tests/simpleStorage/FileStorage.cpp

```diff
@@ -26,9 +26,10 @@ namespace bftEngine {
 FileStorage::FileStorage(Logger &logger, const string &fileName) :
     logger_(logger), fileName_(fileName) {
 
-  dataStream_ = fopen(fileName.c_str(), "wxb+");
```

Comment:
why remove the binary flag?

Reply:
The mode string can also include the letter ``b'' after either the ``+'' or the first letter.  This is strictly for compatibility with ISO/IEC 9899:1990 (``ISO C90'') and has effect only for fmemopen() ; other-
     wise ``b'' is ignored.